src/Core/src/Platform/Windows/WrapperView.cs

```diff
@@ -133,18 +133,18 @@ void UpdateBorder()
 		async partial void ShadowChanged()
 		{
 			if (HasShadow)
-				UpdateShadow();
+				await UpdateShadowAsync();
 			else
 				await CreateShadowAsync();
 		}
 
-		void OnChildSizeChanged(object sender, SizeChangedEventArgs e)
+		async void OnChildSizeChanged(object sender, SizeChangedEventArgs e)
```

Comment:
Pretty risky to have async event handlers since it's an async void, should we ensure that at least we can never throw in there?

Reply:
Yeah, I wasn't sure how we were dealing with async void because I saw we were already doing that on the same file, but Shane just showed me there is a FireAndForget extension method.